### ISSUE TITLE
Cannot read property 'status' of undefined

### ISSUE THREAD

**Issue by phuclh** [64robots/nova-fields#98]
I am using `nova 2.0.5` and `64robots/nova-fields": "^0.8.7`. It show this error when I tried to update an item with Row field (creating is good).

```TypeError: Cannot read property 'status' of undefined```

Moreover, it seems the heading has an issue with CSS. (Nova 2.0 is good)

<img width="522" alt="Screen Shot 2019-06-06 at 4 31 22 PM" src="https://user-images.githubusercontent.com/6707194/59072717-8d532400-8878-11e9-85ee-96ba3f508671.png">

Here is my code:

```
Row::make('Keywords', array(
                        R64Text::make('Keyword')
                            ->fieldClasses('w-full px-8 py-6')
                            ->rules('required', 'max:255')
                            ->hideLabelInForms(),
                        \R64\NovaFields\Number::make('Volume')
                            ->fieldClasses('w-full px-8 py-6')
                            ->rules('numeric', 'min:0')
                            ->hideLabelInForms(),
                        Boolean::make('Is Required')
                            ->hideLabelInForms()
                    )),
```

**Comment by disitec** [comment]
Thanks for the report. This is fixed and will be solved in 0.8.8.

**Comment by disitec** [comment]
BTW UI issues discussed here https://github.com/64robots/nova-fields/issues/100
